src/config.js

```diff
@@ -0,0 +1,9 @@
+import OneGraphAuth from "onegraph-auth";
+
+const APP_ID = "e2ce0bcc-b5b6-42b7-a28e-3a6579d69ecd";
```

Comment:
note: I will need to test this locally. my https://deploy-preview-149--open-sauced.netlify.com/ CORS origin is not enabled.

Reply:
Probably want to move this to an environment variable anyway.